manifests/d/drakkan/SFTPGo/v2.2.1/drakkan.SFTPGo.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: v2.2.1
+Scope: machine
+InstallerSwitches:
+  Silent: "/VERYSILENT"
+  SilentWithProgress: "/SILENT"
```

Comment:
```suggestion
```
These aren't needed here because WinGet will automatically set the `InstallerSwitches` based on `InstallerType`.